olive/passes/olive_pass.py

```diff
@@ -359,7 +359,7 @@ def run(
             components = []
             component_names = []
             for cidx, child in enumerate(model.get_model_components()):
-                component_output_path = Path(output_model_path).with_suffix("") / str(cidx)
+                component_output_path = Path(output_model_path).with_suffix("") / Path(child.model_path).name
```

Comment:
same as the other comment, i think it is better to keep the contents of the components in separate sub directories.

Reply:
We could enumerate over the component names to get a more descriptive sub dir name instead of `str(cidx)`

Reply:
Updated.